src/components/shared/featuredeventcard/featuredeventcard.js

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Day from 'components/shared/day';
+import Box from 'components/shared/box';
+import Text from 'components/shared/text';
+import Button from 'components/shared/button';
+
+import EventPlaceholderImage from 'images/event-placeholder.png';
+
+import ClockIcon from 'images/clock.svg';
+import MapMarkerIcon from 'images/map-marker.svg';
+
+import {
+  Article,
+  Description,
+  ImageWrapper,
+  Image,
+  RSVPBox,
+  IconList,
+  IconRow,
+  IconText,
+  IconLink,
+} from './featuredeventcard.css';
+
+import { parse, format } from 'date-fns';
+
+const FeaturedEventCard = ({ event }) => {
+  const {
+    name,
+    simple_html_description,
+    local_date,
+    time,
+    featured_photo,
+    venue,
+    short_link,
+  } = event;
+
+  const [short_description] = simple_html_description.split('\n');
+  const parsedDateTime = parse(time, 'T', new Date());
+  const formattedTime = format(parsedDateTime, 'h:mm a');
+
+  const image = () => {
+    if (featured_photo) {
+      return <Image src={featured_photo.photo_link} alt="event" />;
+    } else {
+      return <Image src={EventPlaceholderImage} alt="event" />;
+    }
+  };
+
+  const venueLink = () => {
```

Comment:
Would recommend pulling this out into its own component for DRYness since it's duplicated in both cards.

Reply:
Got it. Done via e3e1f9b.